modules/openy_features/openy_location/modules/openy_loc_branch/config/install/field.field.node.branch.field_location_temp_url.yml

```diff
@@ -5,6 +5,7 @@ dependencies:
     - field.storage.node.field_location_temp_url
     - node.type.branch
   module:
+    - datalayer
```

Comment:
can we include https://www.drupal.org/project/datalayer/issues/2991887 patch for not going crazy with multiple unneeded dependencies?
feel free to set RTBC at drupal.org once done